Altis_Life.Altis/core/functions/fn_tazeSound.sqf

```diff
@@ -8,4 +8,4 @@
 private ["_source"];
 _source = [_this,0,objNull,[objNull]] call BIS_fnc_param;
 if (isNull _source) exitWith {};
-_source say3D "tazerSound";
+_source say3D ["tazerSound",20,1];
```

Comment:
The tazerSound maxDistance is 20 metres, but the effective range is 35 metres for the stun pistol and 100 metres for the taser rifle. 

https://github.com/AsYetUntitled/Framework/blob/d8ad6bd7a067d92f23780ad0be637677085c3752/Altis_Life.Altis/core/functions/fn_handleDamage.sqf#L25-L28